### PR TITLE
Adjust histogram buckets and rename reconcile metric

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: Ben Ye <yb532204897@gmail.com>

Adjust the buckets based on the observations on our dashboard.

### CODE REVIEW

File: pkg/localmetrics/localmetrics.go [hunk 1]
```diff
@@ -129,10 +128,10 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			ConstLabels: prometheus.Labels{"name": operatorName},
 		}),
 		reconcileDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
-			Name:        "aws_account_operator_reconcile_duration",
+			Name:        "aws_account_operator_reconcile_duration_seconds",
```
Comment (2uasimojo) [inline pkg/localmetrics/localmetrics.go:131]: ++

File: pkg/localmetrics/localmetrics.go [hunk 2]
```diff
@@ -107,7 +106,7 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			Name:        "aws_account_operator_account_claim_ready_duration_seconds",
 			Help:        "The duration for account claim cr to get claimed",
 			ConstLabels: prometheus.Labels{"name": operatorName},
-			Buckets:     accountDurationBuckets,
+			Buckets:     []float64{1, 5, 10, 20, 30, 45, 60},
```
Comment (rogbas) [inline pkg/localmetrics/localmetrics.go:109]: Considering this is in seconds, I wonder if we should add another bucket for the extreme cases where it takes >1min.

Something like `120`.

Comment (yeya24) [reply pkg/localmetrics/localmetrics.go:109]: That makes sense, will update it.

**Comment by yeya24** [comment]
PTAL again @rogbas @2uasimojo

**Comment by 2uasimojo** [comment]
does
/lgtm
need to be on its own line?

**Comment by rogbas** [comment]
/lgtm
Let's merge this on Monday

**Pull request merged**